src/dependency.ts

```diff
@@ -4,7 +4,7 @@ import { IConstruct } from './construct';
  * Trait marker for classes that can be depended upon
  *
  * The presence of this interface indicates that an object has
- * an `IDependableTrait` implementation.
+ * an `Dependable` implementation.
```

Comment:
I believe this should be
```suggestion
 * an `IDependable` implementation.
```